projects/storefrontlib/src/lib/user/login/login.module.ts

```diff
@@ -1,23 +1,36 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
-import { LoginComponent } from './login.component';
-import { UserModule, UrlTranslationModule } from '@spartacus/core';
-import { CmsModule } from '../../cms/cms.module';
-import { BootstrapModule } from '../../bootstrap.module';
+import {
+  CmsConfig,
+  ConfigModule,
+  UrlTranslationModule,
+  UserModule,
+} from '@spartacus/core';
+// import { CmsModule } from '../../cms/cms.module';
+// import { BootstrapModule } from '../../bootstrap.module';
 import { PageSlotModule } from '../../../cms-structure/page/slot/page-slot.module';
+import { LoginComponent } from './login.component';
 
 @NgModule({
   imports: [
     CommonModule,
     RouterModule,
-    CmsModule,
-    BootstrapModule,
+    // CmsModule,
```

Comment:
Can we remove this ?

Reply:
already gone, probably because of merging with develop branch